src/util/extent.js

```diff
@@ -128,9 +128,9 @@ export default function() {
 
     extents.include = function(x) {
         if (!arguments.length) {
-            return extraPoint;
+            return extraPoints;
         }
-        extraPoint = x;
+        extraPoints = x;
```

Comment:
Yeah, that would be nice - just wondering if there'd be a scenario where you'd want to empty it though, this was mentioned in the original comment in https://github.com/ScottLogic/d3fc/issues/911#issue-136182304